kolibri/utils/compat.py

```diff
@@ -82,3 +82,22 @@ def parse_version(v):
     parsed = _parse_version(v)
 
     return VersionCompat(parsed)
+
+
+def monkey_patch_collections():
```

Comment:
I hate this, but I don't see any alternative whiile we support 2.7, (we don't support < 3.4). One more reason to stop with 2.7 one of these years :( 
My only doubt is if we should use a decorator instead to be applied only to the affected functions instead of the whole module where this patch is applied.

Reply:
They're imported internally to Django, so not really sure how to handle it just as a decorator, unfortunately.